packages/gitgraph-core/src/__tests__/gitgraph.getRenderedData.branches.test.ts

```diff
@@ -221,4 +221,54 @@ describe("Gitgraph.getRenderedData.branches", () => {
       ]);
     });
   });
+
+  describe("deleted branch", () => {
+    let gitgraph;
```

Comment:
🍰 We usually prefer to declare variables closer to their usage, directly into each test. It makes it easier to ensure each test is decoupled from the other (no shared variable).

It's OK here 'cause there are only 2 tests, but if you can refactor this it will be great!

Note that if we want to avoid repeating the same code statements to init the graph state, we can still extract them in a function instead, like so:

```ts
    it("should ignore commits on a deleted branch", () => {
      const { gitgraph, develop } = createGraphWithSomeCommits();

      develop.delete();

      const { commits } = gitgraph._graph.getRenderedData();

      expect(commits).toMatchObject([
        { subject: "one", refs: [] },
        { subject: "two", refs: ["master", "HEAD"] },
      ]);
    });
```